ale_linters/terraform/tflint.vim

```diff
@@ -0,0 +1,44 @@
+" Author: Nat Williams <nat.williams@gmail.com>
+" Description: tflint for Terraform files
+"
+" See: https://www.terraform.io/
+"      https://github.com/wata727/tflint
+
+let g:ale_terraform_tflint_options = get(g:, 'ale_terraform_tflint_options' ,'-f json')
```

Comment:
You can use `ale#Set` for your variables here.

Reply:
`-f json` is essential to the linter, so don't make that part configurable. You could use an empty string by default.